### ISSUE TITLE
onLoadMore will be called incorrectly twice

### ISSUE THREAD

**Issue by tcqq** [davideas/FlexibleAdapter#692]
@davideas `onLoadMore `will be called incorrectly twice when loading more for the first time.

**Comment by tcqq** [comment]
@davideas But this problem means that every time the data is first loaded, 2 pages of data are automatically loaded. This may increase the time it takes to load the data for the first time. Is there any way to overcome this problem now?

**Comment by davideas** [comment]
No, because if you load items that exceeds the page, items are not bound. The loading more is triggered if position exceeds the threshold based on item count.

**Comment by tcqq** [comment]
The page has a total of 10 pieces of data. When the data is loaded for the first time, there are still many items that are not fully displayed on the screen, but in this time it will automatically trigger to load more. 

So it leads to the first time entering the interface, automatically loading 20 pieces of data. 

I feel that this problem may be caused by [mechanism of the no ScrollListener](https://github.com/davideas/FlexibleAdapter/wiki/5.x-%7C-On-Load-More).

**Comment by davideas** [comment]
Negative. It triggers only once for the first 10 items (big enough to exceeds the page) and threshold 1 (default). Then, it waits the last item to be bound to evaluate again the trigger to load more. if your 10 items are all bounds then you should increase the page size. It depends also how long is your screen!

Note: the RV bounds more items than ones are displayed.
